app/session.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html class="no-js" lang="">
+<head>
+  <meta charset="utf-8">
+
+
+  <!-- build:js(.) scripts/iviz-session-vendor.js -->
```

Comment:
Maybe call it vc-session-vendor.js and etc. since it's not part of iViz anymore?